tokenbucket_test.go

```diff
@@ -205,6 +205,51 @@ func (s *LimitersTestSuite) TestTokenBucketRefill() {
 	}
 }
 
+func (s *LimitersTestSuite) TestTokenBucketReset() {
+	for name, backend := range s.tokenBucketBackends() {
+		s.Run(name, func() {
+			clock := newFakeClock()
+
+			bucketCapacity := 2
+			bucket := l.NewTokenBucket(int64(bucketCapacity), time.Second*1, l.NewLockNoop(), backend, clock, s.logger)
+
+			// Check while Bucket is full, partial, empty
+			noOfAccess := []int{0, 1, 2, 3}
```

Comment:
These values are not in use, since in the loop below you are using its index. You can change the slice to `[4, 5, 6, 7]` or to print out `access` and `i` to show that it is the key, not the value of the slice.